python/ray/data/read_api.py

```diff
@@ -791,6 +791,7 @@ def read_images(
     file_extensions: Optional[List[str]] = ImageDatasource._FILE_EXTENSIONS,
     concurrency: Optional[int] = None,
     override_num_blocks: Optional[int] = None,
+    max_image_pixels: Optional[int] = None,
```

Comment:
can you update this arg to something like `pil_image_args`? so we don't need to keep adding new parameters in the future.